Tests/StreamChatTests/MockEnpointResponses/Events/Notification/NotificationMessageNew.json

```diff
@@ -148,7 +148,11 @@
         "mentioned_users" : [
             
         ],
-        "html" : "<p>Hi<\/p>\n"
+        "html" : "<p>Hi<\/p>\n",
+        "pinned": false,
+        "pinned_at": null,
+        "pin_expires": null,
+        "pinned_by": null
```

Comment:
Thank you for updating all payloads! nit: Ideally these changes would be in a separate commit so it's easier to review.